src/view/upcastwriter.js

```diff
@@ -172,6 +172,24 @@ export default class UpcastWriter {
 		return false;
 	}
 
+	/**
+	 * Removes given element from view structure and places its children in its position.
+	 * It does nothing if element has no parent.
+	 *
+	 * @param {module:engine/view/element~Element} element Element which will be unwrapped
+	 * @returns {Booolean} Whether element was successfully unwrapped.
+	 */
+	unwrapElement( element ) {
```

Comment:
Just for my curiosity. Why not just `unwrap`? Other methods from this class (like `remove`, `replace`, `rename`) don't have `Element` suffix.

Reply:
I've followed byt this explanaition:
https://github.com/ckeditor/ckeditor5-paste-from-office/pull/72#discussion_r310022092